### ISSUE TITLE
[MCP] Create designs for reduced expected data for bill history table

### ISSUE THREAD

**Issue by rebeccawalsh** [department-of-veterans-affairs/va.gov-team#29048]
## Issue Description
We learned that the data being sent to us through the CCPC API is much more limited than expected, and we're still trying to figure out what those constraints are. In the meantime, we need to create designs that account for a much more limited dataset.

It's possible that we will only be receiving the data found on the most recent statement, so create designs that account for that.

---
## Tasks
- [x] Design a new detail page depicting the most recent statement's data
- [x] Design a new detail page that includes only the download links
- [x] Upload to UXPin

## Acceptance Criteria
- [x] (2) Designs uploaded to UXPin and linked in this ticket

**Comment by rebeccawalsh** [comment]
**Designs uploaded to UXPin:**
- [Table design for only latest statement data](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/140772652/comment/sitemap?mode=i)
- [Feature cut / no table / download only design](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/141330408?mode=i)

**Next step: **
We're meeting with the liberty team today to determine data availability.

**Comment by LindaLi123** [comment]
hello @rebeccawalsh Please confirm if we are still blocked after yesterday's meeting with liberty team and what additional information you are waiting on

**Comment by rebeccawalsh** [comment]
@LindaLi123 We created this ticket as a blocker to the [design spec work](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/department-of-veterans-affairs/va.gov-team/28015) we were doing, since we had to change the designs to-be-spec'd. We created alternative, limited design mocks with just the most recent statement data AND one without any table at all to accommodate what we thought the data would be after that message from Gayle.

After the meeting yesterday, we basically learned that the work we did was unnecessary because we will receive 6 months of data. As always, we **think** we have an understanding of the data, but this has pretty consistently bit us in the butt. So, in order to do a true spec with certainty, we need to know what the actual data being passed to us. I sent out an email to Mike and Logan (CC'd you, Leah, and Riley) to request mock data with a more accurate data shape so that we can create an accurate spec.

So we will close this ticket since the work is completed, but we will have to make another one in order to update the table and spec with mock data.
